tf_quant_finance/black_scholes/vanilla_prices.py

```diff
@@ -63,12 +66,25 @@ def option_price(volatilities,
     forwards: A real `Tensor` of any shape that broadcasts to the shape of
       `volatilities`. The forwards to maturity. Either this argument or the
       `spots` must be supplied but both must not be supplied.
+    risk_free_rates: An optional real `Tensor` of same dtype as the
+      `volatilities`. If not None, discount factors are calculated using it:
+      e^(-rT), where r is the risk free rate. If not None, `spots` is supplied
+      and `cost_of_carries` is not, also used to compute forwards to expiry.
```

Comment:
How about:
 If not `None`, discount factors are calculated as `e^(-rT)`, where `r` are the risk free rates. 

Not sure if the sentence 'If not None, `spots` is supplied and `cost_of_carries` is not, also used to compute forwards to expiry' is needed. 

Also, please add backticks around `None`. It will make documentation look nicer in Jupyter

Reply:
Is this done?